src/webhooks/pubsub/index.test.ts

```diff
@@ -37,8 +44,8 @@ describe('slack app', function () {
       headers: {
         authorization: 'Bearer 1234abcd',
       },
-    };
-    const reply = new MockReply();
+    } as FastifyRequest<{ Body: { message: { data: string } } }>;
+    const reply = new MockReply() as any;
```

Comment:
What if we cast it as a `FastifyReply` rather than any. Either way it isn't technically right, but at least with `FastifyReply`, we are less broad.

Reply:
The issue is that elsewhere in the code `expect(reply.statusCode).toBe(204);` won't work with the FastifyReply type